src/plugins/intel_gpu/src/graph/impls/ocl/fully_connected.cpp

```diff
@@ -105,6 +135,17 @@ struct fully_connected_impl : typed_primitive_impl_ocl<fully_connected> {
         auto optional_params = get_default_weights_bias_optional_params<kernel_selector::fully_connected_optional_params>(progam);
         optional_params.allowInputReordering = true;
 
+        bool commpressed = !primitive->decompression_scale.empty();
+        bool with_zp = !primitive->decompression_zero_point.empty();
+        if (commpressed) {
+            params.compressed = true;
+            params.decompression_scale = convert_data_tensor(input_layouts[2]);
```

Comment:
If there is bias, shouldn't it be input_layouts[3] ?

Reply:
Not really. `input_layouts` here are not original inputs, but returned from `get_fc_input_layouts` which don't include bias into result